lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/include/Config/PlatformConfig.hpp

```diff
@@ -0,0 +1,262 @@
+
+        /**
+         * \brief The client credentials to use for the platform.
+         */
+        ClientCredentials client_credentials;
+
+        /**
+         * \brief Whether or not the game is a dedicated game server.
+         */
+        bool is_server;
+
+        /**
+         * \brief Result of a bit-wise or operation combining all platform
+         * flags.
+         */
+        int platform_options_flags;
+
+        /**
+         * \brief The AuthScopeFlags for the platform.
+         */
+        EOS_EAuthScopeFlags auth_scope_flags = EOS_EAuthScopeFlags::EOS_AS_NoFlags;
+
+        /**
+         * \brief Integrated platform management flags for the platform.
+         */
+        EOS_EIntegratedPlatformManagementFlags integrated_platform_management_flags = EOS_EIntegratedPlatformManagementFlags::EOS_IPMF_Disabled;
+
+        /**
+         * \brief The maximum amount of execution time the EOS SDK can use each
+         * frame.
+         */
+        int tick_budget_in_milliseconds;
+
+        /**
+         * \brief Indicates the maximum number of seconds that (before first
+         * coming only) the EOS SDK will allow network calls to run before
+         * failing with EOS_TimedOut. This value does not apply after the EOS
+         * SDK has been initialized.
+         */
+        double task_network_timeout_seconds;
+
+        /**
+         * \brief Defines the thread affinity for threads started by the EOS
+         * SDK. Leave values at zero to use default platform settings.
+         */
+        EOS_Initialize_ThreadAffinity thread_affinity;
+
+        /**
+         * \brief If true, the plugin will always send input to the overlay from
+         * the C# side to native side, and handle showing the overlay. This
+         * doesn't always mean input makes it to the EOS SDK.
+         */
+        bool always_send_input_to_overlay;
+
+        /**
+         * \brief Initial button delay (if not set, whatever the default is will
+         * be used).
+         */
+        float initial_button_delay_for_overlay;
+
+        /**
+         * \brief Repeat button delay for the overlay. If not set, whatever the
+         * default is will be used.
+         */
+        float repeat_button_delay_for_overlay;
+
+        /**
+         * \brief Users can press the button(s) associated with this value to
+         * activate the Epic Social Overlay. Not all combinations are valid; the
+         * SDK will log an error at the start of runtime if an invalid
+         * combination is selected.
+         */
+        EOS_UI_EInputStateButtonFlags toggle_friends_button_combination = EOS_UI_EInputStateButtonFlags::EOS_UISBF_None;
+
+        /**
+         * \brief The country code to use when creating the eos platform. Note
+         * that the Unity project does not currently expose this property for
+         * the user to edit. TODO: Expose property in Unity project.
+         */
+        std::string overrideCountryCode;
```

Comment:
Our current implementation within the native code is to _attempt_ to read `overrideCountryCode` and `overrideLocaleCode` from the JSON. In the event that the field cannot be found in the file (which in our case is always because we never write it), a `nullptr` value is assigned. We do not currently provide a mechanism to override this from the command line - whether that be through the native or managed components of the plugin.